Library/Homebrew/cmd/list.rb

```diff
@@ -17,6 +17,9 @@
 #:    If `--pinned` is passed, show the versions of pinned formulae, or only the
 #:    specified (pinned) formulae if <formulae> are given.
 #:    See also `pin`, `unpin`.
+#:
+#:    If `--full-name` is passed, print formulae with fully-qualified names
```

Comment:
This is already noted at the top of the file.

Reply:
Didn't see it. Should it be moved down a line?

Reply:
@SConaway I think it's fine as-is, sorry!